taxcalc/utils.py

```diff
@@ -215,11 +215,11 @@ def count_lt_zero(agg):
 
 
 def weighted_count_lt_zero(agg, col_name):
-    return agg[agg[col_name] < 0]['s006'].sum()
+    return agg[agg[col_name] < -0.0001]['s006'].sum()
 
 
 def weighted_count_gt_zero(agg, col_name):
-    return agg[agg[col_name] > 0]['s006'].sum()
+    return agg[agg[col_name] > 0.0001]['s006'].sum()
```

Comment:
I strongly prefer a `tolerance` keyword arg with a default of `0.0001` or `-0.0001` for both functions. Also, a test function including with something like `weighted_count_gt_zero(agg, col, tolerance=0.004)` or similar would prove it was doing the right thing.